Packs/CiscoASA/Integrations/CiscoASA/README.md

```diff
@@ -234,87 +715,130 @@ Creates a rule.
+| permit | True if the rule is a permit. Possible values are: True, False. | Optional | 
 | destination | The destination. Can be the value of an IPv4, an address block, or the name of a network object. | Optional | 
-| source | The source. Can be the value of an IPv4, an address block, or the name of a network object. object. | Optional | 
+| source | The source. Can be the value of an IPv4, an address block, or the name of a network object. | Optional | 
 | service | The service of the rule. | Optional | 
-
-
-##### Context Output
+| destination_kind | The destination address kind in the ace. Can be on of AnyIPAddress, IPv4Address, IPv4FQDN, IPv4Network, IPv4Range, IPv6Address, IPv6FQDN, IPv6Network, IPv6Range, SecurityName, SecurityTag, interfaceIP, NetworkObj, NetworkObjGroup. Possible values are: AnyIPAddress, IPv4Address, IPv4FQDN, IPv4Network, IPv4Range, IPv6Address, IPv6FQDN, IPv6Network, IPv6Range, SecurityName, SecurityTag, interfaceIP, NetworkObj, NetworkObjGroup. | Optional | 
+| source_kind | The source address kind in the ace. Can be on of AnyIPAddress, IPv4Address, IPv4FQDN, IPv4Network, IPv4Range, IPv6Address, IPv6FQDN, IPv6Network, IPv6Range, SecurityName, SecurityTag, interfaceIP, NetworkObj, NetworkObjGroup. Possible values are: AnyIPAddress, IPv4Address, IPv4FQDN, IPv4Network, IPv4Range, IPv6Address, IPv6FQDN, IPv6Network, IPv6Range, SecurityName, SecurityTag, interfaceIP, NetworkObj, NetworkObjGroup. | Optional | 
+| service_kind | The source service kind, can be one of AnyService, ICMP6Service, ICMPService, NetworkProtocol, NetworkServiceGroups, NetworkServiceObjects, TcpUdpService. Possible values are: AnyService, ICMP6Service, ICMPService, NetworkProtocol, NetworkServiceGroups, NetworkServiceObjects, TcpUdpService. | Optional | 
+| destination_service | The destination service value. | Optional | 
+| destination_service_kind | The destination service kind, can be one of AnyService, ICMP6Service, ICMPService, NetworkProtocol, NetworkServiceGroups, NetworkServiceObjects, TcpUdpService. Possible values are: AnyService, ICMP6Service, ICMPService, NetworkProtocol, NetworkServiceGroups, NetworkServiceObjects, TcpUdpService. | Optional | 
+| time_range | Object ID of for the time range object. Can be received with the command cisco-asa-list-time-range. | Optional | 
+| user | The object ID to the user; can be one of LocalUserObjGroup, UserGroupObj, UserObj. Can be received with existing commands; cisco-asa-list-local-user-group, cisco-asa-list-local-user, cisco-asa-list-user-object. | Optional | 
+| user_kind | The type of the user, can be one of LocalUserObjGroup, UserGroupObj, UserObj. Possible values are: LocalUserObjGroup, UserGroupObj, UserObj. | Optional | 
+| source_security | Source security group in the ace. Can be one of SecurityName, SecurityTag and SecurityObjGroup values can be added. Can be received with the existing command cisco-asa-list-security-object-group. | Optional | 
```

Comment:
Don't understand this sentence.
"Can be one of SecurityName, SecurityTag and SecurityObjGroup values can be added."
Please clarify.